libpod/container_internal.go

```diff
@@ -1014,6 +1025,51 @@ type resolvConf struct {
 	options       []string
 }
 
+// generatePasswd generates a container specific passwd file,
+// iff g.config.User is a number
+func (c *Container) generatePasswd() (string, error) {
+	var (
+		groupspec string
+		gid       uint32
+	)
+	if c.config.User == "" {
```

Comment:
Can we reuse the chrootuser parse code here? We don't want the extra chroot, but the parsing should be identical?